### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by mohitdhatrak**
Change log: Added wishlist page

Page is responsive too.

image

image

image

### CODE REVIEW

**Review by poojakelkar — COMMENTED** [review]
Looks Good to merge , Nice Design

File: Styles/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+.wishlist-heading {
+    margin-bottom: 0;
+    margin-top: 1.5em;
+    text-align: center;
+    text-transform: uppercase;
+}
+
+.card-container {
+    padding: 0;
+    margin: 2em 0;
```
Comment (tushargoyal812) [inline Styles/wishlist.css:43]: suggestion: use consistent units like if you are using rem use rem everywhere.

**Review by tushargoyal812 — COMMENTED** [review]
just take what is suggested otherwise looks good to merge.

File: Styles/wishlist.css (hunk 1 above)
Comment (mohitdhatrak) [reply Styles/wishlist.css:43]: Ok, actually i am using rem for font size width and em for padding margin gap as i want them to scale with font size, which mostly wont happen as i wont change so rem em doesn't make a difference here

**Comment by mohitdhatrak** [comment]
> Looks Good to merge , Nice Design

Thanks

**Pull request merged**